lib/pharos/phases/configure_master.rb

```diff
@@ -7,6 +7,8 @@ class ConfigureMaster < Pharos::Phase
 
       KUBE_DIR = '/etc/kubernetes'
       SHARED_CERT_FILES = %w(ca.crt ca.key sa.key sa.pub front-proxy-ca.key front-proxy-ca.crt).freeze
+      APISERVER_CERT = '/etc/kubernetes/pki/apiserver.crt'
```

Comment:
Should probably use `KUBE_DIR`.

Reply:
IMO using the consts like that just makes the code less clear than defining the full path. Should I change it?

Reply:
Maybe it's fine, we can refactor this later if it still feels bad.